src/EnergyPlus/HeatBalanceSurfaceManager.cc

```diff
@@ -5289,7 +5289,7 @@ CalcHeatBalanceInsideSurf( Optional_int_const ZoneToResimulate ) // if passed in
 			// check for saturation conditions of air
 			Real64 const HConvIn_surf( HConvInFD( SurfNum ) = HConvIn( SurfNum ) );
 			RhoVaporAirIn( SurfNum ) = min( PsyRhovFnTdbWPb_fast( MAT_zone, ZoneAirHumRat_zone, OutBaroPress ), PsyRhovFnTdbRh( MAT_zone, 1.0, HBSurfManInsideSurf ) );
-			HMassConvInFD( SurfNum ) = HConvIn_surf / ( ( PsyRhoAirFnPbTdbW_fast( OutBaroPress, MAT_zone, ZoneAirHumRat_zone ) + RhoVaporAirIn( SurfNum ) ) * PsyCpAirFnWTdb_fast( ZoneAirHumRat_zone, MAT_zone ) );
+			HMassConvInFD( SurfNum ) = HConvIn_surf / ( PsyRhoAirFnPbTdbW_fast( OutBaroPress, MAT_zone, ZoneAirHumRat_zone ) * PsyCpAirFnWTdb_fast( ZoneAirHumRat_zone, MAT_zone ) );
```

Comment:
@Myoldmopar<https://github.com/Myoldmopar>, the equation below was changed because the previous equation was incorrect. The specific heat (PsyCpAir...) is defined on a dry-air basis [J/kg_da-K], so you want to multiply only by the density of air, (PsyRhoAir...) [kg_da/m^3]. Adding to this the vapor density (RhoVaporAirIn) puts the density on a total air/vapor mixture basis [kg_total/m^3], which is inconsistent with how the specific heat is defined, and thus gives the incorrect answer.


From: Noel Merket [mailto:notifications@github.com]
Sent: Tuesday, August 09, 2016 8:52 AM
To: NREL/EnergyPlus
Cc: Woods, Jason; Mention
Subject: Re: [NREL/EnergyPlus] Improved Effective Moisture Penetration Depth model (#5093)


In src/EnergyPlus/HeatBalanceSurfaceManager.cc<https://github.com/NREL/EnergyPlus/pull/5093#discussion_r74073928>:

> @@ -5289,7 +5289,7 @@ CalcHeatBalanceInsideSurf( Optional_int_const ZoneToResimulate ) // if passed in

>                      // check for saturation conditions of air

>                      Real64 const HConvIn_surf( HConvInFD( SurfNum ) = HConvIn( SurfNum ) );

>                      RhoVaporAirIn( SurfNum ) = min( PsyRhovFnTdbWPb_fast( MAT_zone, ZoneAirHumRat_zone, OutBaroPress ), PsyRhovFnTdbRh( MAT_zone, 1.0, HBSurfManInsideSurf ) );

> -                    HMassConvInFD( SurfNum ) = HConvIn_surf / ( ( PsyRhoAirFnPbTdbW_fast( OutBaroPress, MAT_zone, ZoneAirHumRat_zone ) + RhoVaporAirIn( SurfNum ) ) * PsyCpAirFnWTdb_fast( ZoneAirHumRat_zone, MAT_zone ) );

> +                    HMassConvInFD( SurfNum ) = HConvIn_surf / ( PsyRhoAirFnPbTdbW_fast( OutBaroPress, MAT_zone, ZoneAirHumRat_zone ) * PsyCpAirFnWTdb_fast( ZoneAirHumRat_zone, MAT_zone ) );

@Myoldmopar<https://github.com/Myoldmopar>, if I had to guess, I would say this is the culprit. Maybe I'll revert this one change and see if it gets rid of the ESO small diffs.

@jdwoods21<https://github.com/jdwoods21>, Do you remember why we did this? Was it a bug fix?

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/NREL/EnergyPlus/pull/5093/files/6ff1ecf82271c50cad2ef95fdb365875882cdac1#r74073928>, or mute the thread<https://github.com/notifications/unsubscribe-auth/ANCL-wWWKPNrHSD16TGTrZsAMUJGM2bHks5qeJQMgaJpZM4FqclD>.